Homework14/src/test/java/IUATest.java

```diff
@@ -0,0 +1,134 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class IUATest {
+    private WebDriver driver;
+
+    public static final String LOGIN = "Taras1-2";
+    public static final String PASSWORD = "Test1234";
+    public static final String EMAIL = "Taras1-2@i.ua";
+    public static final String SUBJECT = "Изучение Автомейшен";
+    public static final String BODY = "Я рад, что решился пойти на курсы!";
+
+
+    @BeforeMethod
+    public void setUp() {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+        driver.get("https://www.i.ua/");
+    }
+
+    @AfterMethod
+    public void tearDown() {
+        driver.close();
+        driver.quit();
+    }
+
+    @Test(priority = 1)
+    public void loginIntoSite(){
+        WebElement searchLoginPage = driver.findElement(By.cssSelector("body > div.Branding_header.page_medium.clear > div.Header.clear > ul.user_menu > li.first > a"));
```

Comment:
Please try to simplify selector, e.g. use sorter one like "ul.user_menu > li.first > a"
Also searchLoginPage variable name does not match with the meaning